### PR TITLE
[py] build(setup.py): Add `project_urls` for PyPI

### PR DESCRIPTION
**Pull request by tony**
### Description

Show Project URLs to [Selenium's PyPI page](https://pypi.org/project/selenium/) via [`project_urls`](https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#project-urls) to _setup.py_

image

### Motivation and Context

It takes a few clicks to get around to the docs and source code.  This makes it possible launch all of them via PyPI. 

It also is helpful to anyone else that ingests the metadata from setup.py / PyPI

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by tony** [comment]
@symonk Do you have a take on this?

File: py/setup.py [hunk 1]
```diff
@@ -32,6 +32,12 @@
     'description': 'Python bindings for Selenium',
     'long_description': open(join(abspath(dirname(__file__)), "README.rst")).read(),
     'url': 'https://github.com/SeleniumHQ/selenium/',
+    'project_urls': {
+        'Bug Tracker': 'https://github.com/SeleniumHQ/selenium/issues',
+        'Changes': 'https://github.com/SeleniumHQ/selenium/blob/trunk/py/CHANGES',
```
Comment (symonk) [inline py/setup.py:37]: is this better pointing at: `https://github.com/SeleniumHQ/selenium/blob/trunk/py/CHANGES` specifically?

File: py/setup.py [hunk 2]
```diff
@@ -32,6 +32,12 @@
...
+        'Documentation': 'https://seleniumhq.github.io/selenium/docs/api/py/api.html',
```
Comment (symonk) [inline py/setup.py]: I think this is old docs, tho I'm a little out of the know on the docs atm, maybe it should pointing at: https://www.selenium.dev/documentation/overview/?

File: py/setup.py (hunk 1 above)
Comment (tony) [reply py/setup.py:37]: @symonk That's the same URL? Is there a different one you'd prefer?

File: py/setup.py (hunk 2 above)
Comment (tony) [reply py/setup.py]: Updated in 4664fe02cc

File: py/setup.py (hunk 1 above)
Comment (symonk) [reply py/setup.py:37]: apologies missed the `/py/`

**Comment by symonk** [comment]
test failures here are likely flakes at the minute, `bidi` tests need some TLC.  fine to merge

**Comment by symonk** [comment]
Thanks @tony for the contribution, much appreciated.  Apologies for the delays on review!

**Comment by tony** [comment]
You're welcome, Thank you @symonk !

**Pull request merged**